tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+
+When scanning through the rest of this code, we don't see this string literal being used anywhere obviously, so they will pause on that question for now.
+
+The second argument seems to be a large, anonymous arrow function with zero parameters, that contains even more unfamiliar syntax.
+
+<!-- prettier-ignore-start -->
+```js
+() => {
+    // ...
+}
+```
+<!-- prettier-ignore-end -->
+
+Let's approach the contents of this function piece-by-piece.
+
+### Hypothesizing About `test`
+
+Inside the anonymous function started in `describe`, there is a block of code that begins with `test()`.
+
+<!-- prettier-ignore-start -->
+```js
+  test('returns Fizz on multiples of 3', () => {
+    // ...
+  });
+```
+<!-- prettier-ignore-end -->
+
+This is also a function being invoked. This `test` function seems to take in two arguments, the string literal `'returns Fizz on multiples of 3'` and an anonymous arrow function.
+
+Leila notices that this pattern feels very similar to the `describe` block. 🤔 They should keep that observation, because it's a good one!
+
+### Arrange
+
+Inside the next anonymous arrow function, we see the following code:
+
+<!-- prettier-ignore-start -->
+```js
+    // Arrange
+    const num = 9;
+```
+<!-- prettier-ignore-end -->
+
+It's a comment indicating an "Arrange" step and a `const` variable `num` assigned to the number `9`.
+
+Leila is an overthinker, and they could dig deeper into what this line means. However, this time, they will choose to make only one theory: the variable `num` will be used later in the test.
+
+### Act
+
+We can move on to reading the next set of lines:
+
+<!-- prettier-ignore-start -->
+```js
+    // Act
+    const result = fizzBuzz(num);
+```
+<!-- prettier-ignore-end -->
+
+Leila recognizes this as the "Act" step in a test, where the behavior we're testing (a `fizzBuzz` function) should be "acted on," or invoked.
+
+Here, we are invoking `fizzBuzz`, passing in `num`, and saving the return value into a `const` variable named `result`.
+
+### Assert
+
+Let's dive into the final line, which features some more unfamiliar syntax.
+
+<!-- prettier-ignore-start -->
+```js
+    // Assert
+    expect(result).toEqual('Fizz');
+```
+<!-- prettier-ignore-end -->
+
+Leila knows that the assert step is the step that determines whether a test passes or fails.
+
+They've seen the `test` block include the string "returns Fizz on multiples of 3," which Leila suspects is relevant to the Assert step of this test.
+
+Reading the line from left-to-right, it seems like `expect(result)` is also a function call. The function's name is `expect` and we're passing in `result`.
+
+However, the line doesn't stop there. Leila sees `.toEqual('Fizz')`. Recalling other JavaScript syntax, Leila suspects that this is a function call, where the function is named `toEqual` and the argument is `'Fizz'`.
+
+It looks like this function call operates from an object. That object must be `expect(result)`.
```

Comment:
`expect(result)` is not itself the object, but calling this returns some object that we use.
add a bit where Leila is ok with not knowing exactly what this object is, but moves forward with their deduction.
```suggestion
It looks like this function call operates on an object. That object must be whatever is returned from the call to `expect(result)`. Leila doesn't know exactly what this object is, but they can tell that it must have a method called `toEqual`!
```